### PR TITLE
Update various links

### PR DESCRIPTION
**Pull request by atouchet**

### CODE REVIEW

**Comment by ashawley** [comment]
Thanks for all these fixes!

**Comment by ashawley** [comment]
There's a very strange error on the Travis build:
```
$ grep johno/pixyll    _site/css/pixyll.css
grep: _site/css/pixyll.css: No such file or directory
The command "grep johno/pixyll    _site/css/pixyll.css" exited with 2.
```

File: _config.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ email:       your_email@example.com
 author:      John Otander
 description: "A simple, beautiful theme for Jekyll that emphasizes content rather than aesthetic fluff."
 baseurl:     ""
-url:         "http://pixyll.com"
+url:         "https://pixyll.com/"
```
Comment (ashawley) [inline _config.yml:7]: I don't think Jekyll is expecting the trailing slash.  The https is fine.

**Comment by ashawley** [comment]
> There's a very strange error on the Travis build:

I though this was a spurious error, but I can reproduce locally when I check out the branch:

```
$ grep johnotander/pixyll _site/css/pixyll.css
grep: _site/css/pixyll.css: No such file or directory
```

fwiw I haven't run bundle, so it's not from a gem update.

File: _sass/_base.scss [hunk 2]
```diff
@@ -1,15 +1,15 @@
-@charset "UTF-8";
+﻿@charset "UTF-8";
```
Comment (ashawley) [inline _sass/_base.scss:1]: I'm staring at this, but I'm not sure what the change is on this line.  Maybe it's responsible for the broken build?

Comment (ashawley) [reply _sass/_base.scss:1]: Looking at `git diff` in the console it seems it's `<U+FEFF>`

File: css/pixyll.scss [hunk 3]
```diff
@@ -1,13 +1,13 @@
----
+﻿---
```
Comment (ashawley) [inline css/pixyll.scss:1]: Same issue repeats itself here.

**Review by ashawley — APPROVED** [review]

File: _sass/_base.scss (hunk 2 above)
Comment (atouchet) [reply _sass/_base.scss]: Not sure what happened there. Thanks for taking care of it.

**Pull request merged**